Sources/Sentry/Public/SentrySpanProtocol.h

```diff
@@ -62,6 +67,11 @@ NS_SWIFT_NAME(Span)
 - (void)setDataValue:(nullable id)value
               forKey:(NSString *)key NS_SWIFT_NAME(setExtra(value:key:));
 
+/**
+ * Sets a tag value.
+ */
+- (void)setTagValue:(nullable id)value forKey:(NSString *)key NS_SWIFT_NAME(setTag(value:key:));
```

Comment:
`h`: As we are going to change the `nullable id` to `NSString` we also need a `removeTag`, like in the Scope: https://github.com/getsentry/sentry-cocoa/blob/fab5383cd946400112e8124b74429baed30302f5/Sources/Sentry/Public/SentryScope.h#L34-L37